### ISSUE TITLE
Map auto delete

### ISSUE THREAD

**Issue by hup333** [osmandapp/OsmAnd#10740]
# 🚀 feature request

### Describe the solution you'd like

 If you have a solution in mind, please describe it.

### Describe alternatives you've considered

 Have you considered any alternative solutions or workarounds?

**Comment by vshcherb** [comment]
hm... interesting case, we have it only for raster maps, so far

**Comment by hakuchi** [comment]
A better solution I think is a pop up that show you older maps. Including activated oes and deactivatet.

**Comment by sonora** [comment]
Yes, a map should not simply  'expire'. There must always be a user consent to delete a file you had once deliberately downloaded.
